src/frontends/tensorflow/include/openvino/frontend/tensorflow/decoder.hpp

```diff
@@ -11,6 +11,8 @@ namespace ov {
 namespace frontend {
 namespace tensorflow {
 
+class TENSORFLOW_API EmptyList {};
```

Comment:
Can we put this inside `DecoderBase` class or hide it anywhere else? Not sure it should be on the same level as `DecoderBase`

Reply:
I wouldn't recommend to use nested class here (inside DecoderBase I mean).
Why is it TENSORFLOW_API?

Reply:
OVTF would use it, I guess they would need API header

Reply:
maybe we can move it to separate file, e.g. special_types.hpp